src/app/components/layout/two-column.layout.tsx

```diff
@@ -0,0 +1,43 @@
+import { Flex, styled } from 'leather-styles/jsx';
+
+interface TwoColumnLayoutProps {
+  leftColumn: React.JSX.Element;
+  rightColumn: React.JSX.Element;
+}
+//  TODO this can be a more generic panda layout component
+export function TwoColumnLayout({
+  leftColumn,
+  rightColumn,
+}: TwoColumnLayoutProps): React.JSX.Element {
+  return (
+    <Flex
+      flexDirection={['column', 'column', 'column', 'row']}
+      paddingTop="space.06"
+      gap="space.05"
+    >
+      {/* left column */}
+      <Flex
+        alignItems={['center', 'center', 'center', 'flex-start']}
+        textAlign={['center', 'center', 'center', 'left']}
+        flexDirection="column"
+        padding="space.00"
+        gap="space.07"
+        width={['100%', '500px']}
+        mx={['auto', 'auto', 'auto', 'unset']}
+      >
+        <styled.div maxWidth="440px">{leftColumn}</styled.div>
+      </Flex>
+
+      {/* right column */}
+      <Flex
+        gap="space.05"
+        alignItems={['center', 'center', 'center', 'flex-start']}
+        flexDirection="column"
+        px={['space.08', 'space.08', 'space.08', 'space.02']}
+        width={['100%', '600px']}
+      >
+        {rightColumn}
+      </Flex>
+    </Flex>
+  );
+}
```

Comment:
This looks pretty mnemonic page specific. Is it going to be reusable elsewhere? E.g. there's a fixed width that maybe in another two-column layout we don't want.

Reply:
I'm not sure where I will put this in the end but the same layout is shared on 4 pages now so I thought it worth re-using it. 

We need this on all mnemonic keys pages and also the sign in form that shows there

Reply:
I kept this but moved into the `components/secret-key` folder. Hopefully that's OK? 

I think we could make a panda container for this and for our other main view which is one centered content. 

I could even try and use their existing patterns - `VStack` and `Center`